### ISSUE TITLE
Security issue

### ISSUE THREAD

**Issue by lirantal** [danlevan/bracket-template#21]
Hi,

I'm a member of the Node.js Security WG and we received a report regarding a security issue with this module.

I couldn't locate any e-mail address associated with the author of this module so I'm opening this issue and inviting anyone with commit and npm publish rights to collaborate with us on a fix.

**Comment by DanielRuf** [comment]
`git log` should provide these information (except the email address is kept private in the profile settings).

```
git show b06390cc37f89d356e3e0fc01de3f7d54e49f988
```

```
Merge: b82a3d5 ecfe2f5
Author: danlevan <...@gmail.com>
Date:   Sun Jan 8 19:01:44 2017 -0500

    Bump version
```

**Comment by lirantal** [comment]
Indeed. Have since gone public: https://github.com/nodejs/security-wg/blob/9eacac1c03348531ce3fcffbda1cf04d9d0394f4/vuln/npm/404.json
